### ISSUE TITLE
Colored text should be made more accessible

### ISSUE THREAD

**Issue by guidezpl** [flutter/website#6486]
### Describe the problem

The accessibility of the blue and green text used throughout the site could be improved

<img width="714" alt="Screenshot 2021-11-18 at 17 17 22" src="https://user-images.githubusercontent.com/6655696/142454109-3577c407-f9c3-42f6-ae1c-1dedb55fd44e.png">
For example

<img width="225" alt="Screenshot 2021-11-18 at 17 19 36" src="https://user-images.githubusercontent.com/6655696/142454347-d346ec7f-652a-4350-8885-82970201af0a.png">

### Expected fix

Darken the colors used

**Comment by tomByrer** [comment]
I think there are new [contrast color guidelines coming out](https://github.com/w3c/silver/issues/285#issuecomment-783776854), but I can't find automation for it ATT.

**Comment by Myndex** [comment]
OOPS forgot, here's the npm install, import, and useage:


## QuickStart
### _Install_

```javascript
    npm i apca-w3
```

### _Import_
```javascript
    import { APCAcontrast, sRGBtoY, displayP3toY, colorParsley } from 'apca-w3';
```
### _Usage:_
PARSE:
If you need to parse a color string or 24bit number, use the colorParsley() function:
```javascript
    let rgbaArray = colorParsley('aliceblue');
```
CONVERT TO Ys
Send rgba INT array [123,123,123,1.0] to sRGBtoY() — this is a slightly different luminance Y that the IEC oiecewise.

```javascript
    let Ys = sRGBtoY([123,123,123,1.0]);
```
FIND Lc CONTRAST
First color _must_ be text, second color must be the background.

```javascript
    let textColor = [17,17,17,255];
    let backgroundColor = [232,230,221,255];
    
    let contrastLc = APCAcontrast( sRGBtoY( textColor ), sRGBtoY( backgroundColor ) );
```
## _String Theory_
The following are the available input types for colorParsley(), HSL is not implemented at the moment. All are automatically recognized:

#### INPUT as STRINGS:
- **No Alpha**
    - ` '#abc' ` or ` 'abc' ` (interpreted as ` 'aabbcc' `)
    - ` '#abcdef' ` or ` 'abcdef' ` (hash is ignored)
    - ` 'rgb(123, 45, 67)' `
    - ` 'aquamarine' ` or ` 'magenta' ` (full CSS4 named colors list)

- **With Alpha** _(alpha is NOT presently calculated, and assumed as fully opaque)_
    - ` '#abcf' ` or ` 'abcf' ` (interpreted as ` 'aabbccff' `)
    - ` '#123456ff' ` or ` '123456ff' ` (hash is ignored)
    - ` 'rgba(123, 45, 67,255)' `

#### INPUT as NUMBER:
- **As hex**
    - ` 0xabcdef `
- **As integer**
    - ` 11259375 `

No alpha parsing for _numbers_ in part as there are big and little endian issues that need to be resolved.

### Parsing Removal
The function is called "colorParsley()" because what is that useless leafy thing the restaurant puts on the plate?  Well, most mature software already has good parsing, and you may want to minimize the file leaving all that "parsley" at the restaurant.

In the src folder .js file, there is a ` /*/ ` type code toggle, see the comments just before the parsing functions. you can disable the entire set of parsing functions by removing a slash before minimizing if you like to go lean and clean.

This changes the import you need to use to:

```javascript
             // import with parsing off/removed:
    import { APCAcontrast, sRGBtoY, displayP3toY } from 'apca-w3';
```


### Font Use Lookup Table
Latest Lookup Table: November 17 2021

<img width="639" alt="0.0.98G4gLUT" src="images/0.0.98G4gLUT.png">

<img width="596" alt="0.0.98G4gLUT legend" src="images/0.0.98G4gLUT-legend.png">

```javascript
// APCA FONT LOOKUP TABLE 0.98G-4g-b3
// Font Size and Reference Font Weight
// THIS GRID FOR FLUENT TEXT USE CASE ONLY DEC 12 2021

const apcaFluentGrid = [
   ["min", "min", "min", "min", "min", "min", "min", "min", "min"],
   ["min", "min", "min", "min", "min", "min", "min", "min", "min"],
   ["min", "min", "min", 95, 90, 85, 80, "min", "min"],
   ["min", "min", "min", 90, 85, 80, 75, "min", "min"],
   ["min", "min", 95, 80, 75, 65, 60, 55, "min"],
   ["min", "min", 90, 75, 65, 60, 55, 50, 45],
   ["min", 95, 85, 65, 60, 55, 50, 45, 40],
   ["min", 90, 75, 60, 55, 50, 45, 40, 35],
   ["min", 85, 70, 55, 50, 45, 40, 35, 30],
   [90, 75, 60, 50, 45, 40, 35, 30, "max"],
   [85, 70, 55, 45, 40, 35, 30, "max", "max"],
   [75, 60, 50, 40, 35, 30, "max", "max", "max"],
   [70, 55, 45, 35, 30, "max", "max", "max", "max"],
   [60, 45, 40, 30, "max", "max", "max", "max", "max"],
 ];
```

Thank you!

Andy

**Comment by atsansone** [comment]
This issue is being addressed in https://github.com/flutter/website/issues/9427. Closing this issue as a duplicate.
